### DISCUSSION TITLE
[NOTICE]: deprecating Node v12 and moving to Node v16

### DISCUSSION THREAD

**Discussion by mittalyashu** [logchimp/logchimp#566] [General]
This is a prior notice, that we're planning to move to Node v16 (not anytime soon) but that on the roadmap.

**What's included:**
* [LogChimp](https://github.com/logchimp/logchimp)
* [Theme](https://github.com/logchimp/theme) _(will eventually moved to monorepo)_
* [CLI](https://github.com/logchimp/logchimp-cli)

We'll try to make the transition from Node v12 to v16 for you as smooth as possible without any breaking changes and will keep you posted via this thread.

**Comment by ChrisLally** [comment]
Any updates on this @mittalyashu? - I did get a related error when trying to deploy

**Reply by mittalyashu** [reply]
Hi @clally1,

We'll be using Node v16 in LogChimp v0.7. If you're facing issue deploying the v0.7 (beta). I suggest you try deploying LogChimp v0.6.x

Can you also share the error you're getting while deploying?
